### PR TITLE
refactor: Sync with master and JDK 11 update (RDCC-1622)

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1622

### Change description ###
refactor: Sync with master and JDK 11 update (RDCC-1622)

**Does this PR introduce a breaking change?** (check one with "x")
No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ JRD is batch application and JRD batch is scheduled with Kubernetes which runs o
 JRD consume data files from an external source, transform that data into the destination format 
 and load the data into JRD database. 
 
+JRD Uses Data-Ingestion-Library (https://github.com/hmcts/data-ingestion-lib) and configure camel routes with
+data-ingestion-lib & JRD custom configuration files & crates camel integration framework which reads CSV files & Stores
```
Comment (shreedhar-hmcts) [inline README.md:10]: you can mention the sample gradle entry of lib ,will be easy for any new person to implement

Comment (abhijitdiwan) [reply README.md:10]: done.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -33,59 +26,50 @@ spring:
...
+archival-path: azure-blob://${azure.storage.account-name}/jud-ref-data-archive
+active-blob-path: azure-blob://${azure.storage.account-name}/jud-ref-data
+archival-file-names: Personal,Appointments,Roles,Contracts,Locations,BaseLocations
+archival-cred: credentials=#credsreg&operation=updateBlockBlob
+archival-date-format: dd-MM-yyyy--HH-mm
+file-read-time-out: 180000
+leaf-route-task: leafRouteTask
+parent-route-task: parentRouteOrchestrationTask
+batchjob-name: JudicialDataLoad
+scheduler-audit-select: select count(*) from dataload_schedular_audit where date(scheduler_start_time) = current_date
```
[the remark this answers is not in the document]
Comment (abhijitdiwan) [reply src/main/resources/application.yaml]: done.

File: src/main/java/uk/gov/hmcts/reform/juddata/camel/util/JrdMappingConstants.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    public static final String DATE_PATTERN_TIMESTAMP = "\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2}.\\d{4,9}";
+
+    public static final String DATE_TIMESTAMP_FORMAT = "yyyy-MM-dd HH:MI:SS.MSUS";
+
+    public static final String DATE_TIME_FORMAT = "yyyy-MM-dd hh:mm:ss";
+
+    public static final String ORCHESTRATED_ROUTE = "parent-route";
+
+    public static final String LEAF_ROUTE = "leaf-route";
+
```
Comment (shashank-rastogi) [inline src/main/java/uk/gov/hmcts/reform/juddata/camel/util/JrdMappingConstants.java:20]: leaf route should be renamed to judicial-ref-data-leaf-orchestration

**Comment by shashank-rastogi** [comment]
Java Doc comments needs to be updated on all the files

File: src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/JrdBatchIntegrationSupport.java [hunk 4]
```diff
@@ -6,12 +6,11 @@
...
-import uk.gov.hmcts.reform.juddata.camel.service.AuditProcessingService;
-import uk.gov.hmcts.reform.juddata.camel.service.EmailService;
-import uk.gov.hmcts.reform.juddata.camel.util.DataLoadUtil;
+import uk.gov.hmcts.reform.data.ingestion.camel.processor.ExceptionProcessor;
+import uk.gov.hmcts.reform.data.ingestion.camel.route.DataLoadRoute;
+import uk.gov.hmcts.reform.data.ingestion.camel.service.IEmailService;
+import uk.gov.hmcts.reform.data.ingestion.camel.util.DataLoadUtil;
+import uk.gov.hmcts.reform.juddata.camel.service.JudicialAuditServiceImpl;
 
 public abstract class JrdBatchIntegrationSupport {
```
Comment (shashank-rastogi) [inline src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/JrdBatchIntegrationSupport.java:16]: Java Doc comments needs to be updated on all the files

**Pull request closed without merging**